docs/release_notes/8_8.md

```diff
@@ -53,80 +53,83 @@ Some highlights and new features from this release are detailed next.
 
 ### Security
 
-Security-related highlights in the latest Rocky Linux 8.8 release are listed below. For a complete list of security related changes, see the [upstream link here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.8_release_notes/new-features#new-features-security).
-
 * FIPS mode settings in the kernel have been adjusted to conform to the Federal Information Processing Standard (FIPS) 140-3. This change introduces stricter settings to many cryptographic algorithms, functions, and cipher suites.
```

Comment:
Does the docs team have a guideline where to use sentences and where just bulletpoints? IMO bulletpoints are no sentences.